virttest/asset.py

```diff
@@ -494,9 +494,13 @@ def download_file(asset_info, interactive=False, force=False):
         else:
             answer = 'y'
         if answer == 'y':
-            download.url_download_interactive(url, destination,
-                                              "Downloading %s" % title)
-            had_to_download = True
+            try:
+                download.url_download_interactive(url, destination,
+                                                  "Downloading %s" % title)
+                had_to_download = True
+            except Exception, download_failure:
+                logging.error("Check your internet connection: %s",
```

Comment:
I do not understand the logic. JeOS file is not a requirement to enable VT Plugins, right? The user has the option to avoid the download. So, if the download fails, why will need to raise the exception?